server/api/controllers/job.py

```
@@ -0,0 +1,270 @@
+                {
+                    "jobs": jobs_data,
+                    "total": total,
+                    "page": page,
+                    "limit": limit,
+                    "total_pages": total_pages,
+                }
+            ),
+            200,
+        )
+
+    except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)